cogs/highlight/highlight.py

```diff
@@ -239,6 +241,82 @@ def check(msg):
                                          "currently".format(userName))
             await self._sleepThenDelete(confMsg, 5)
 
+    @highlight.group(name="ignore")
+    @commands.guild_only()
+    async def wordIgnore(self, ctx: Context):
+        """Ignore certain words to avoid having them trigger your DMs.
+
+        Suppose you have a word X in your highlighted words, and a word Y you are
+        ignoring.  Then, we have some scenarios as below:
+        - "X something something": triggers DM.
+        - "X Y": does NOT triggers DM.
+        - "X something something Y": does NOT trigger DM.
+        """
+
+    @wordIgnore.command(name="add")
+    @commands.guild_only()
+    async def wordIgnoreAdd(self, ctx: Context, *, word: str):
+        """Add words to your ignore list.
+
+        Parameters:
+        -----------
+        word: str
+            The word you wish to ignore.
+        """
+        userName = ctx.message.author.name
+
+        async with self.config.member(ctx.author).ignoreWords() as ignoreWords:
+            if len(ignoreWords) < MAX_WORDS and word not in ignoreWords:
+                ignoreWords.append(word)
+                confMsg = await ctx.send("{} added to the ignore list, "
+                                         "{}".format(word, userName))
+            else:
+                confMsg = await ctx.send("Sorry {}, you are already ignoring {} "
+                                         "words, or you are trying to add a "
+                                         "duplicate word".format(userName,
+                                                                 MAX_WORDS))
```

Comment:
Slight nitpick here, but maybe we want a differnet constant for blacklisted words and highlgihted words